cluster-provision/okd/hacks/release-4.4

```diff
@@ -1,34 +1,3 @@
-diff --git a/cmd/openshift-install/create.go b/cmd/openshift-install/create.go
-index f9ae4c6bb..dea45f0d7 100644
---- a/cmd/openshift-install/create.go
-+++ b/cmd/openshift-install/create.go
-@@ -244,7 +244,7 @@ func waitForBootstrapComplete(ctx context.Context, config *rest.Config, director
- 
- 	discovery := client.Discovery()
- 
--	apiTimeout := 30 * time.Minute
-+	apiTimeout := 120 * time.Minute
- 	logrus.Infof("Waiting up to %v for the Kubernetes API at %s...", apiTimeout, config.Host)
- 	apiContext, cancel := context.WithTimeout(ctx, apiTimeout)
- 	defer cancel()
-@@ -285,7 +285,7 @@ func waitForBootstrapComplete(ctx context.Context, config *rest.Config, director
- // and waits for the bootstrap configmap to report that bootstrapping has
- // completed.
- func waitForBootstrapConfigMap(ctx context.Context, client *kubernetes.Clientset) error {
--	timeout := 30 * time.Minute
-+	timeout := 120 * time.Minute
```

Comment:
are you sure its ok ? 
we do need to extend timeout, please see #301 suggestion

did you manage with it to finish provision + cluster-up of 4.3 ? 4.4 ?
trying on 4.4 , i am not sure that it will finish the provision

Reply:
If this works out for you, let's go for that.
I didn't manage provisioning in the end.

Reply:
i am still working on it, focusing on 4.4 first
thanks